docs/sources/flow/setup/install/kubernetes.md

```diff
@@ -59,5 +57,14 @@ For more information on the Grafana Agent Helm chart, refer to the Helm chart do
 
 ## Next steps
 
-- [Start Grafana Agent]({{< relref "../start-agent" >}})
-- [Configure Grafana Agent]({{< relref "../configure/configure-kubernetes" >}})
+<!--- [Start Grafana Agent][]-->
```

Comment:
should we remove the commented out unused link here and below?

Reply:
I was going to, but I wanted to double check that Agent in K8s deployments doesn't need an explicit start - in my local test it doesn't.

Reply:
Removed it :-)